src/core/components/pin/gc-lock.js

```diff
@@ -0,0 +1,135 @@
+'use strict'
+
+const mortice = require('mortice')
+const pull = require('pull-stream')
```

Comment:
Please import just the things you need from `pull-stream` like we do with `async`.